docs/source/examples/workspaces/random/PinFile.jenkins.j2

```diff
@@ -0,0 +1,36 @@
+jenkins-slave:
+  topology:
+    topology_name: jenkins-slave
+    resource_groups:
+    - resource_group_name: beaker-slaves
+      resource_group_type: beaker
+      resource_definitions:
+      - role: bkr_server
+        job_group: "{{ job_group }}"
+        whiteboard: "{{ arch }} slave for multiarch testing"
+        recipesets:
+        - distro: RHEL-7.4
+          arch: "{{ arch }}"
+          variant: Server
+          count: 1
+  layout:
+    inventory_layout:
+      vars:
+        hostname: __IP__
+      hosts:
+        example-node:
+          count: 1
+          host_groups:
+            - example
+  {% if hooks %}
+  hooks:
+    {% if hooks.postUp %}
+    post-up:
+      {% if hooks.postUp.connectToMaster %}
+      name: cinch
+      type: shell
+      actions:
+        - cinch inventories/jenkins-slave.inventory
+      {% endif %}
+    {% endif %}
+  {% endif %}
```

Comment:
Can this go in a `jenkins-slave` workspace?

Reply:
yes